### DISCUSSION TITLE
Web UI Timeouts

### DISCUSSION THREAD

**Discussion by yzislin** [gravitational/teleport#13930] [Q&A] [answered]
Hello,
I am using Teleport with Servers and Applications configuration in place. I have a user who is curious about the following.
He is using Web UI to gain access to Servers (ie SSH). When he is idle, his laptop goes to sleep. Later on, when it wakes up, his session gets kicked out. 

Two questions:
1. Is it possible to preserve a web ui session through hibernation? Maybe have like 10 hours of forced time out.
2. Is it possible to resume SSH session where left off?

Thank you.

**Comment by webvictim** [comment, marked as answer]
No to both. For security, the web UI's authentication uses a bearer token which is only valid for a few minutes and is refreshed on a loop. If the laptop goes to sleep and doesn't refresh the token, the session will expire.

If the connection is interrupted then it will need to be re-established. Teleport does not support mechanisms like `mosh` today.

**Reply by yzislin** [reply]
Thank you for detailed explanation.
